### PR TITLE
Extract getAvatarPathWithInitialsFallback, use in ApeAvatar and AMForceGraph

### LINKED ISSUE #272: Change ApeAvatar component to default to user's initials
**Current Behavior:**
The ApeAvatar component currently defaults to a [placeholder image](https://github.com/coordinape/coordinape/blob/master/src/components/ApeAvatar/ApeAvatar.tsx#L22), which isn't very informative.

**Desired Behavior:**
Update the [ApeAvatar component](https://github.com/coordinape/coordinape/blob/master/src/components/ApeAvatar/ApeAvatar.tsx) to display the user's initials instead of the placeholder image.

### PR DESCRIPTION
**Pull request by zencephalon**
Closes #272 

This extracts placeholder logic from `ApeAvatar` in `util/domains` and uses the new logic in both `ApeAvatar` and `AMForceGraph`.

Builds on work from https://github.com/coordinape/coordinape/pull/278

### CODE REVIEW

File: src/utils/domain.ts [hunk 1]
```diff
@@ -62,3 +62,14 @@ export const getAvatarPath = (avatar?: string, placeholder?: string) => {
 
   return `${STORAGE_URL}/${avatar}`;
 };
+
+export const getAvatarPathWithInitialsFallback = (
```
Comment (levity) [inline src/utils/domain.ts:66]: how about `getAvatarPathWithFallback` for brevity's sake? also a bit more future-proof if we change the implementation

Comment (zencephalon) [reply src/utils/domain.ts:66]: Yeah, sounds good to me.

**Comment by levity** [comment]
hmm, I'm seeing this on the preview app with Chrome/macOS:

Screen Shot 2021-11-10 at 3 07 18 PM

**Comment by zencephalon** [comment]
@levity Mm, yeah, that looks wrong. I'm finding it a bit hard to setup a circle where I can test this feature on my own. Could I get added to a circle in staging so I can try it out there?

**Comment by zencephalon** [comment]
Screen Shot 2021-11-11 at 3 58 06 PM

Hey @levity thanks for taking a look yesterday. I believe I have it working today.

Also cool that U also enjoy martial arts and transpersonal psychology! I feel curious about whether we've read some of the same books because that strikes me as a particular combination of interests.

**Review by levity — APPROVED** [review]
nice!

**Comment by levity** [comment]
> Also cool that U also enjoy martial arts and transpersonal psychology! I feel curious about whether we've read some of the same books because that strikes me as a particular combination of interests.

I bet we have! :)

**Pull request merged**